CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# dbt_reddit_ads v1.3.0
+[PR#31](https://github.com/fivetran/dbt_reddit_ads/pull/31) includes the following updates:
+
+## Schema/Data Change
+**12 total changes • 0 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ---------- | ----------- | -------- | -------- | ----- |
+| All reddit_ads__*_report models | New Column |  | `purchase_conversions` | Total attributed click-through conversions for purchase events. |
+| All reddit_ads__*_report models | New Column |  | `purchase_view_through_conversions` | Total attributed view-through conversions for purchase events. |
+| All reddit_ads__*_report models | New Column |  | `purchase_value` | Total monetary value associated with purchase conversion events. |
+| All reddit_ads__*_report models | New Column |  | `purchase_items` | Total number of items involved in purchase conversion events. |
+| All reddit_ads__*_report models | New Column |  | `lead_conversions` | Total attributed click-through conversions for lead generation events. |
+| All reddit_ads__*_report models | New Column |  | `lead_view_through_conversions` | Total attributed view-through conversions for lead generation events. |
+| All reddit_ads__*_report models | New Column |  | `lead_value` | Total monetary value associated with lead generation conversion events. |
+| All reddit_ads__*_report models | New Column |  | `lead_items` | Total number of items involved in lead generation conversion events. |
+| All reddit_ads__*_report models | New Column |  | `custom_conversions` | Total attributed click-through conversions for custom-defined events. |
+| All reddit_ads__*_report models | New Column |  | `custom_view_through_conversions` | Total attributed view-through conversions for custom-defined events. |
+| All reddit_ads__*_report models | New Column |  | `custom_value` | Total monetary value associated with custom-defined conversion events. |
+| All reddit_ads__*_report models | New Column |  | `custom_items` | Total number of items involved in custom-defined conversion events. |
+
+By default, the package includes conversion metrics for three event types: `purchase`, `lead`, and `custom`. You can configure alternative conversion events to be tracked by adjusting the `reddit_ads__conversion_event_types` variable in your `dbt_project.yml`. Each event type configured will generate four corresponding columns (`{event_type}_conversions`, `{event_type}_view_through_conversions`, `{event_type}_value`, and `{event_type}_items`) across all report models.
```

Comment:
could be good to link to the README [section](https://github.com/fivetran/dbt_reddit_ads?tab=readme-ov-file#configure-conversion-event-types)